VinylTests/Arbitrary.swift

```diff
@@ -13,7 +13,7 @@ import SwiftCheck
 let lowerStringGen =
 Gen<Character>.fromElements(in: "a"..."z")
     .proliferateNonEmpty
-    .map(String.init)
+    .map { String($0) }
```

Comment:
`$0` is `[Character]` so `String($0)` is using this initialiser: `init<S>(_ characters: S) where S : Sequence, S.Element == Character`

However, when referenced as a function as previously, the compiler cannot infer which `init` to use, so we have to do it this way 🤷🏼‍♀️